File: Android/framework/src/test/java/CustomerServices/Customer.java
package CustomerServices;

import com.IdentityFieldAnnotation;
import com.DataManipulation.BaseMapperInterfaces.IBaseMapper;
import com.Domain.DomainObject;

/**
 * Created by aiko on 3/5/17.
 */

public class Customer extends DomainObject {
    public Customer(IBaseMapper mapper)
    {
        super(mapper);
    }

    @IdentityFieldAnnotation
    public String Number;

    public String Name;
}

File: DotNet/CSharp/Framework.Tests/CustomerServices/Customer.cs
﻿using Framework.Data_Manipulation;
using Framework.Domain;

namespace Framework.Tests.CustomerServices
{
    public class Customer : DomainObject
    {
        public class InnerClass { }

        public Customer(IBaseMapper mapper, IBaseQueryObject queryObject) : base(mapper, queryObject)
        {
        }

        [IdentityField]
        public string Number { get; set; }

        public string Name { get; set; }

        [IgnorePropertyMonitoring]
        public bool NotSoHelpfulMember { get; set; }

        public InnerClass Temp { get; set; }
    }
}
